[2024-08-07T09:06:11.313+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-07T09:06:11.331+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: update_11str.update_11str scheduled__2024-08-07T04:00:00+00:00 [queued]>
[2024-08-07T09:06:11.337+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: update_11str.update_11str scheduled__2024-08-07T04:00:00+00:00 [queued]>
[2024-08-07T09:06:11.338+0000] {taskinstance.py:2303} INFO - Starting attempt 2 of 2
[2024-08-07T09:06:11.348+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): update_11str> on 2024-08-07 04:00:00+00:00
[2024-08-07T09:06:11.356+0000] {standard_task_runner.py:63} INFO - Started process 504 to run task
[2024-08-07T09:06:11.359+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'update_11str', 'update_11str', 'scheduled__2024-08-07T04:00:00+00:00', '--job-id', '32', '--raw', '--subdir', 'DAGS_FOLDER/str11_batch.py', '--cfg-path', '/tmp/tmp7fsu0bxq']
[2024-08-07T09:06:11.362+0000] {standard_task_runner.py:91} INFO - Job 32: Subtask update_11str
[2024-08-07T09:06:11.401+0000] {task_command.py:426} INFO - Running <TaskInstance: update_11str.update_11str scheduled__2024-08-07T04:00:00+00:00 [running]> on host 36f576e8f7f1
[2024-08-07T09:06:11.467+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='zzioni' AIRFLOW_CTX_DAG_ID='update_11str' AIRFLOW_CTX_TASK_ID='update_11str' AIRFLOW_CTX_EXECUTION_DATE='2024-08-07T04:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-07T04:00:00+00:00'
[2024-08-07T09:06:11.468+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-07T09:06:12.317+0000] {logger.py:11} INFO - ====== WebDriver manager ======
[2024-08-07T09:06:12.343+0000] {logger.py:11} INFO - Get LATEST chromedriver version for google-chrome
[2024-08-07T09:06:12.344+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-07T09:06:12.345+0000] {taskinstance.py:2890} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/str11_batch.py", line 26, in update_11str
    price, prices_id, deleted = extract_price_11str()
  File "/opt/airflow/dags/utils/crawl_11str.py", line 102, in extract_price_11str
    driver = webdriver.Chrome(service=Service(ChromeDriverManager().install()), options=chrome_options)
  File "/home/airflow/.local/lib/python3.10/site-packages/webdriver_manager/chrome.py", line 40, in install
    driver_path = self._get_driver_binary_path(self.driver)
  File "/home/airflow/.local/lib/python3.10/site-packages/webdriver_manager/core/manager.py", line 40, in _get_driver_binary_path
    file = self._download_manager.download_file(driver.get_driver_download_url(os_type))
  File "/home/airflow/.local/lib/python3.10/site-packages/webdriver_manager/drivers/chrome.py", line 32, in get_driver_download_url
    driver_version_to_download = self.get_driver_version_to_download()
  File "/home/airflow/.local/lib/python3.10/site-packages/webdriver_manager/core/driver.py", line 48, in get_driver_version_to_download
    return self.get_latest_release_version()
  File "/home/airflow/.local/lib/python3.10/site-packages/webdriver_manager/drivers/chrome.py", line 64, in get_latest_release_version
    determined_browser_version = ".".join(determined_browser_version.split(".")[:3])
AttributeError: 'NoneType' object has no attribute 'split'
[2024-08-07T09:06:12.359+0000] {taskinstance.py:1205} INFO - Marking task as FAILED. dag_id=update_11str, task_id=update_11str, execution_date=20240807T040000, start_date=20240807T090611, end_date=20240807T090612
[2024-08-07T09:06:12.382+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 32 for task update_11str ('NoneType' object has no attribute 'split'; 504)
[2024-08-07T09:06:12.412+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-08-07T09:06:12.451+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-07T09:06:12.456+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
